README.md

```diff
@@ -19,16 +19,11 @@
 Plone backend [Docker](https://docker.com) images using Python 3 and [pip](https://pip.pypa.io/en/stable/).
 
 > **NOTE**:
-> These images are not officially supported by the Plone Community for Plone 5.
-> They will become the official ones for the upcoming Plone 6 release (together with [plone.frontend](https://github.com/plone/plone-frontend)).
-> These image are **not** buildout based!
+> These images are the official ones for the Plone 6 release (together with [plone.frontend](https://github.com/plone/plone-frontend)).
+> These image are **not** Buildout based!
```

Comment:
This begs the question, what are they based on? Or do you mean "not compatible with buildout"?

```suggestion
These images are **not** Buildout based!
```